apps/prairielearn/src/pages/authCallbackLti/authCallbackLti.js

```diff
@@ -71,7 +73,7 @@ router.post('/', function (req, res, next) {
       // https://oauth.net/core/1.0/#nonce
       var nonceReused = false;
       var nonceKey = 'authCallbackLti:' + parameters.oauth_timestamp + ':' + parameters.oauth_nonce;
-      cache.get(nonceKey, (err, val) => {
+      util.callbackify(cache.get)(nonceKey, (err, val) => {
```

Comment:
Rather than putting callback garbage in the cache implementation itself, I'm pushing the callbackification out to this one place that it's actually used. I think we should do this in general as we convert from callbacks to promises.

Reply:
I'm on board!